packages/perseus/src/widgets/interactive-graphs/stateful-mafs-graph.tsx

```diff
@@ -109,7 +109,12 @@ export const StatefulMafsGraph = React.forwardRef<
     const numPoints = graph.type === "point" ? graph.numPoints : null;
     const numSides = graph.type === "polygon" ? graph.numSides : null;
     const snapTo = graph.type === "polygon" ? graph.snapTo : null;
-    const showAngles = graph.type === "polygon" ? graph.showAngles : null;
+    const showAngles =
+        graph.type === "polygon" || graph.type === "angle"
+            ? graph.showAngles
+            : null;
+    const allowReflexAngles =
+        graph.type === "angle" ? graph.allowReflexAngles : null;
```

Comment:
These options needed to be added in order to update the graph previews while editing these options in the Content Editor.